projects/isy-angular-widgets/assets/theme/_isyfact-variables.scss

```diff
@@ -60,13 +60,14 @@ $accordionHeaderPadding: 0.875rem !default;
 //accordion
 $accordionSpacing: 0px !default;
 
-// Variables in theme file _variables.scss adjusted to meet color requirements of the Isyfact design for the input field
+// Variables in theme file _variables.scss adjusted to meet color requirements of the IsyFact design
 
-////input
+//font-weight
+$isyfact-bold: bold !default;
+
+//input
 $inputPlaceholderTextColor: #83878c !default;
 
 //progress spinner
-.p-progress-spinner-circle {
-  stroke: $themePrimary;
-  animation: p-progress-spinner-dash 1.5s ease-in-out infinite;
-}
+$isyfact-progress-spinner-stroke: $themePrimary !default;
+$isyfact-progress-spinner-animation: p-progress-spinner-dash 1.5s ease-in-out infinite !default;
```

Comment:
$isyfactProgressSpinnerAnimation: p-progress-spinner-dash 1.5s ease-in-out infinite !default;

Reply:
Done